docs/getting_started/index.md

```diff
@@ -172,48 +82,41 @@ Here's a simple GUI set up for our "Hello World" scenario:
         the Page Builder API.
 
         ```python linenums="1"
-        import taipy as tp
+        from taipy.gui import Gui 
         import taipy.gui.builder as tgb
+        from math import cos, exp
 
-        # Previous configuration of scenario
-        ...
-
-        with tgb.Page() as page:
-            tgb.text("Name:")
-            tgb.input("{input_name}")
-            tgb.button("Submit", on_action=submit_scenario)
-            tgb.text("Message {message}")
-
+        value = 10
 
-        input_name = "Taipy"
-        message = None
+        def compute_data(decay:int)->list:
+            return [cos(i/6) * exp(-i*decay/600) for i in range(100)]
 
+        def on_slider(state):
+            state.data = compute_data(state.value)
 
-        def submit_scenario(state):
-            state.scenario.input_name.write(state.input_name)
-            state.scenario.submit(wait=True)
-            state.message = scenario.message.read()
+        with tgb.Page() as page:
+            tgb.text(value="Taipy Getting Started", class_name="h1")
+            tgb.text(value="Value: {value}")
+            tgb.slider(value="{value}", on_change=on_slider)
+            tgb.chart(data="{data}") 
 
+        data = compute_data(value)
 
-        if __name__ == "__main__":
-            tp.Core().run()
-            scenario = tp.create_scenario(scenario_cfg)
-            tp.Gui(page).run()
+        Gui(page=page).run(title="Frontend Demo")
         ```
 
-        [Get the whole code](./hello_world_tgb.py){: .tp-btn target='blank'}
-
         Now, let’s explain the key elements of this code:
 
         ```python
         import taipy as tp
+        import taipy.gui.builder as tgb
 
 
         with tgb.Page() as page:
-            tgb.text("Name:")
-            tgb.input("{input_name}")
-            tgb.button("Submit", on_action=submit_scenario)
-            tgb.text("Message {message}")
+            tgb.text(value="Taipy Getting Started", class_name="h1")
```

Comment:
or `tgb.html("h1", "Taipy Getting Started")` which I think is better

Reply:
That would be wrong, since you rely on the h1 style. Not Taipy.
Do as you wish.
To me this makes a lot more sense, just like in Gradio or Streamlit: Taipy has support for HTML tags.